code/modules/mob/living/say.dm

```diff
@@ -252,7 +252,7 @@ var/list/channel_to_radio_key = new
 
 	message = trim_left(message)
 	var/message_range = world.view
-	if(!(speaking && (speaking.flags & NO_STUTTER)))
+	if(!skip_edit && !(speaking && (speaking.flags & NO_STUTTER)))
```

Comment:
why do you need a separate variable instead of just checking if `whisper` is true

Reply:
because when calling verb whisper it will need to use this (using say :w has it go through a different path that would make this trigger twice)